### ISSUE TITLE
All messages learned as `LOCAL_FUZZY_UNKNOWN:1`

### ISSUE THREAD

**Issue by moisseev** [vstakhov/rspamd#675]
~~~
# rspamc -w 20 -f 13 fuzzy_add bill.eml
Results for file: bill.eml (0.022 seconds)
success = true;
~~~
~~~
2016-06-22 11:39:53 #70961(controller) <6b1d62>; csession; rspamd_controller_check_password: allow unauthorized connection from a trusted IP 127.0.0.1
2016-06-22 11:39:53 #70961(controller) <46b3a9>; task; rspamd_message_parse: loaded message; id: <1466584546.431377702@f324.i.mail.ru>; queue-id: <undef>
2016-06-22 11:39:53 #70961(controller) <46b3a9>; task; fuzzy_controller_io_callback: processed fuzzy hash 2e5c00b57e, list: LOCAL_FUZZY_UNKNOWN:1 for message <1466584546.431377702@f324.i.mail.ru>
2016-06-22 11:39:53 #70961(controller) <46b3a9>; task; fuzzy_controller_io_callback: processed fuzzy hash 2b01e4cab3, list: LOCAL_FUZZY_UNKNOWN:1 for message <1466584546.431377702@f324.i.mail.ru>
2016-06-22 11:39:53 #70961(controller) <46b3a9>; task; fuzzy_controller_io_callback: processed fuzzy hash 5fddfa21e6, list: LOCAL_FUZZY_UNKNOWN:1 for message <1466584546.431377702@f324.i.mail.ru>
2016-06-22 11:39:53 #70961(controller) <46b3a9>; task; fuzzy_controller_io_callback: processed fuzzy hash 560c84d5a3, list: LOCAL_FUZZY_UNKNOWN:1 for message <1466584546.431377702@f324.i.mail.ru>
2016-06-22 11:39:53 #70961(controller) <46b3a9>; task; fuzzy_controller_io_callback: processed fuzzy hash ee9761b434, list: LOCAL_FUZZY_UNKNOWN:1 for message <1466584546.431377702@f324.i.mail.ru>
~~~
~~~
# rspamadm configdump fuzzy_check
2016-06-22 11:42:20 #76136((NULL)) <6fjagz>; lua; dmarc.lua:279: cannot parse servers parameter
*** Section fuzzy_check ***
min_bytes = 100;
rule {
    rspamd.com {
        servers = "rspamd.com:11335";
        encryption_key = "icy63itbhhni8bq15ntp5n5symuixf73s1kpjh6skaq4e7nx5fiy";
        symbol = "FUZZY_UNKNOWN";
        mime_types [
            "application/*",
        ]
        max_score = 20.0;
        read_only = true;
        skip_unknown = true;
        fuzzy_map {
            FUZZY_DENIED {
                max_score = 20.0;
                flag = 1;
            }
            FUZZY_PROB {
                max_score = 10.0;
                flag = 2;
            }
            FUZZY_WHITE {
                max_score = 2.0;
                flag = 3;
            }
        }
    }
}
rule {
    local {
        servers = "127.0.0.1:11335";
        symbol = "LOCAL_FUZZY_UNKNOWN";
        mime_types = "*";
        max_score = 20.0;
        read_only = false;
        skip_unknown = true;
        fuzzy_map {
            LOCAL_FUZZY_DENIED {
                max_score = 20.0;
                flag = 11;
            }
            LOCAL_FUZZY_PROB {
                max_score = 10.0;
                flag = 12;
            }
            LOCAL_FUZZY_WHITE {
                max_score = 2.0;
                flag = 13;
            }
        }
    }
}
*** End of section fuzzy_check ***
~~~

**Comment by moisseev** [comment]
> is this issue reproduced when there is merely one fuzzy rule defined?

Nothing's changed.
~~~
*** Section fuzzy_check ***
min_bytes = 100;
rule {
    local {
        servers = "127.0.0.1:11335";
        symbol = "LOCAL_FUZZY_UNKNOWN";
        mime_types = "*";
        max_score = 20.0;
        read_only = false;
        skip_unknown = true;
        fuzzy_map {
            LOCAL_FUZZY_WHITE {
                max_score = 2.0;
                flag = 13;
            }
        }
    }
}
*** End of section fuzzy_check ***
~~~
~~~
2016-06-22 20:59:00 #98731(controller) <0d3821>; task; fuzzy_controller_io_callback: processed fuzzy hash 1e82f3bcb4, list: LOCAL_FUZZY_UNKNOWN:1 for message <eae5ce42-078c-a4bb-6a5e-5036df719d6c@goszakaz93.ru>
~~~

**Comment by moisseev** [comment]
Fixed with 0e09a5486513847905fb059185a7c5a32f50ead9
